apps/frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,15 @@
+import { iconNameToSvg } from "./libs/maps/icon-name-to-svg.map.js";
+import { type IconName } from "./types/icon-name.type.js";
+
+interface IconProperties {
```

Comment:
I don't know if it is an issue, but we define `type`s for props, and name them Properties